run.sh

```diff
@@ -1,22 +1,46 @@
+#!/bin/bash
+
 # note: must be run from the project root folder
 
+# set env variables
+envFile="$PWD/.env"
+source "$envFile"
+BOOTSTRAP_SERVER=$BOOTSTRAP_SERVER
+DEPOSIT_GROUP=$DEPOSIT_GROUP
+DEPOSIT_TOPIC=$DEPOSIT_TOPIC
+DEPOSIT_BUCKET_NAME=$DEPOSIT_BUCKET_NAME
+DEPOSIT_KEY_NAME=$DEPOSIT_KEY_NAME
+K_AWS_ACCESS_KEY_ID=$K_AWS_ACCESS_KEY_ID
+K_AWS_SECRET_ACCESS_KEY=$K_AWS_SECRET_ACCESS_KEY
+K_AWS_SESSION_TOKEN=$K_AWS_SESSION_TOKEN
+K_AWS_EXPIRATION=$K_AWS_EXPIRATION
+HEADER_ACCEPT=$HEADER_ACCEPT
+HEADER_X_API_KEY=$HEADER_X_API_KEY
+API_ENDPOINT=$API_ENDPOINT
+KAFKA_TYPE=$KAFKA_TYPE
+CONFLUENT_KEY=$CONFLUENT_KEY
+CONFLUENT_SECRET=$CONFLUENT_SECRET
```

Comment:
Is this necessary if we are already sourcing from the env file?

Reply:
These have been updated to be set as environment variables - the updated run.sh does require that these values are set as environment variables so that they can be accessed by the s3-depositor